src/concrete/ml/quantization/quantizers.py

```diff
@@ -62,9 +63,16 @@ def fill_from_kwargs(obj, klass, **kwargs):
     # If the structure was created or modified by a call to this function, check
     # that it is completely filled
     if obj is not None:
-        for name in hints:
-            if getattr(obj, name) is None:
-                raise TypeError(f"Missing quantizer parameter {name}")
+        all_members_missing = all(getattr(obj, name) is None for name in hints)
```

Comment:
I wanted the `stats` to be optional -> to be able to create a quantizer out of analytically determined values, e.g. when no min/max is computed.

But I want the caller to either be able to use all the fields in `MinMaxStatistics` or none of them.